src/ExportPro.Common/ExportPro.Common.Shared/Library/NotFoundResponse.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Net;
+
+
+namespace ExportPro.Common.Shared.Library;
+
+public class NotFoundResponse : BaseResponse
+{
+    public NotFoundResponse(bool isSuccess = false, string message = null)
```

Comment:
I prefer to go with the empty constructor. we can directly map the message or isSuccess value after creating a class

Reply:
just have empty constructor where you are mapping status code